### PR TITLE
Release

### PR DESCRIPTION
**Pull request by jschueller**
If you want to submit something before the 1.11rc1, speak now :)

### CODE REVIEW

**Comment by regislebrun** [comment]
+1

File: ChangeLog [hunk 1]
```diff
@@ -89,12 +89,10 @@
  * #926 (Covariance model active parameter set behavior)
  * #928 (Covariance model/ Field/Process properties naming)
  * #932 (Optim algo iteration/evaluation number)
- * #933 (Compilation failed when Muparser is missing)
  * #937 (Small bugs in FunctionalChaosResult::get{Residuals, RelativeErrors})
  * #938 (DistributionFactory::buildEstimator to handle Exception)
  * #939 (The help of the getMaximumDegreeStrataIndex method is wrong.)
  * #941 (Duplicate with SobolIndicesExperiment)
- * #943 (Sample don't show up correctly in notebooks)
```
Comment (dbarbier) [inline ChangeLog]: Is this change intentional (removal of two bugs)?

File: distro/debian/patches/series [hunk 2]
```diff
@@ -1,4 +1,2 @@
 tests-disable-coupling_tools.patch
 discard-levelsetmesher-test.patch
-fix-multiprocessing-hurd.patch
-test-NormalGamma.patch
```
Comment (dbarbier) [inline distro/debian/patches/series]: I will have to keep test-NormalGamma.patch in Debian

File: ChangeLog (hunk 1 above)
Comment (jschueller) [reply ChangeLog]: Yes, these two bugs affected the git version only, not a published version, so the end user does not care about that.

File: distro/debian/patches/series (hunk 2 above)
Comment (jschueller) [reply distro/debian/patches/series]: Why is that ? I applied it.

File: ChangeLog [hunk 3]
```diff
@@ -89,12 +89,10 @@
...
  * #944 (Inline plots crash in notebooks if too many points)
  * #942 (Negative optimization bounds in Kriging)
```
Comment (sofianehaddad) [inline ChangeLog]: @jschueller I think 942 also affect git version ?

Comment (jschueller) [reply ChangeLog]: Indeed
